deposit/src/main/java/edu/unc/lib/deposit/validate/ExtractTechnicalMetadataJob.java

```diff
@@ -138,13 +138,16 @@ public void runJob() {
 
                 // Store the premis report to file
                 writePremisReport(objPid, premisDoc);
+                addClicks(1);
             } catch (JobFailedException | JobInterruptedException e) {
                 throw e;
             } catch (Exception e) {
                 failJob(e, "Failed to extract FITS details for {0} from document:\n{1}",
                         objPid, xmlOutputter.outputString(fitsDoc));
             }
         }
+
+        setTotalClicks(stagingList.size());
```

Comment:
For some of the longer running jobs it'd be helpful to know in advance how much work there is total. 
There are some examples of precomputing the count, this is from the virus scan job:
```
List<Entry<PID, String>> hrefs = getPropertyPairList(model, CdrDeposit.stagingLocation);

        setTotalClicks(hrefs.size());
```

I think both the jobs you updated in this PR would be good candidates